doc/source/license.rst

```diff
@@ -7,11 +7,7 @@ License
 
 OPEN SOURCE LICENSE AGREEMENT
 =============================
-Creative Commons license (CC BY 4.0)
-
-Copyright (c) 2024, The Trustees of Columbia University in
-the City of New York.
-All Rights Reserved.
+Creative Commons Zero v1.0 Universal license (CC0 1.0)
```

Comment:
OK -- I looked more carefully in LICENSE.rst and it seems to be CC0 license (public domain, copyright waived?), so edited it and removed the copyright lines also... Let me know if this is okay.

Currently in LICENSE.rst, there's a legal explanation of CC0 but not specific text that says "this package is under CC0..." --> do we add anything there?

At the bottom of LICENSE.rst there's also a line that says "NOTE: ace and jsoneditor in the regolith/static/ dir are covered under their own licences."
^What are these licenses and do we need to add this line to doc/source/license.rst also?

Reply:
I think that just leaving the text as is there is ok.

Also for the CC0 I think just leaving it as it is is fine.  I think it is understood that the license in license.rst is the license that applies to the package.    If you want, you could look around in other packages.  It also probably doesn't do harm to add "this package is covered by....".